src/Simulation/Simulators/ToffoliSimulator/Processor.cs

```diff
@@ -0,0 +1,429 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using static System.Math;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Quantum.Simulation.Common;
+using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.Simulators.Exceptions;
+
+namespace Microsoft.Quantum.Simulation.Simulators
+{
+
+    // ToffoliSimulator is implemented based on `QuantumProcessorDispatcher`,
+    // which is constructed using a specialization `QuantumProcessorBase`.  The
+    // specialization overrides methods to specify actions on intrinsic
+    // operations in the Q# code.
+    class ToffoliSimulatorProcessor : QuantumProcessorBase {
```

Comment:
> { [](start = 58, length = 2)

Please follow the coding style of the project and place opening braces on new lines.

Reply:
Am going to fix it in one of the next commits.